### DISCUSSION TITLE
Cloud support for pg_graphql extension

### DISCUSSION THREAD

**Discussion by florianlouvet** [supabase/supabase#6506] [General]
Hey guys,

I've been following the progress made on the pg_graphql extension to postgres for a while. Before the latest release in Beta, the documentation was containing some sentence stating that in the future it might be possible to install the extension on postgres database hosted at cloud providers such as AWS or Gcloud or others,...
This sentence disappeared, and I can't find any information regarding if this is still something that is planned or already possible. If anyone can help I would greatly appreciate it.

Thanks all for the amazing supabase work done around here!

**Comment by dinvlad** [comment]
Same question here, AWS RDS and GCP Cloud SQL don't seem to allow installation of unapproved extensions. Are there any plans to apply for approvals? Or is this extension purely SQL-based and thus doesn't need an approval?
